### DISCUSSION TITLE
Unable to build CC: Tweaked Documentation site on gradle

### DISCUSSION THREAD

**Discussion by SkyTheCodeMaster** [cc-tweaked/CC-Tweaked#716] [Q&A] [answered]
So, I'm trying to add a pastebin module to CC:T. I have the module, and when I add it to my local copy of the CC:T github repo, I can lint it fine, with no errors/warnings from it.
I'm trying to build the documentation site to see how the documentation looks, but when I do `./gradlew docWebsite`, it fails within 5 seconds withn the error `sh: 1: node_modules/.bin/rollup: not found`. I don't know what I'm doing wrong, as the illuaminate lint runs fine.
![image](https://user-images.githubusercontent.com/34724753/107808392-806a2180-6d37-11eb-8504-338860dfafea.png)

**Comment by SkyTheCodeMaster** [comment]
Forgot to include the link to the pastebin module! sorry!
https://gist.github.com/SkyTheCodeMaster/12aeb6cd7d2890640bd81bab66261a4b

**Comment by SquidDev** [comment, marked as answer]
See https://github.com/SquidDev-CC/CC-Tweaked/blob/mc-1.15.x/CONTRIBUTING.md#setting-up-the-tooling.
